src/Qir/Execution/QirCommandLineTool/QirCommandLineTool.csproj

```diff
@@ -1,15 +1,43 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="..\..\..\Simulation\Common\Simulators.Dev.props" />
+    
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <PackAsTool>true</PackAsTool>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.20213.1" />
-    <ProjectReference Include="..\..\..\Simulation\Simulators\Microsoft.Quantum.Simulators.csproj" />
-    <ProjectReference Include="..\Tools\Microsoft.Quantum.Qir.Tools.csproj" />
+    <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.20213.1"/>
+    <ProjectReference Include="..\Tools\Microsoft.Quantum.Qir.Tools.csproj"/>
+    <PackageReference Include="Microsoft.Quantum.Qir.Runtime" Version="0.16.210526824-alpha"/>
```

Comment:
Yes, though it requires that the QIR Runtime has built already. If you take a look at the build script:
https://github.com/microsoft/qsharp-runtime/blob/85bbb3eaa4755510d224ae53cc5acaa65e956fa1/src/Qir/Runtime/build-qir-runtime.ps1#L18-L28
it includes logic to copy into the drops folder, as used by the e2e build to collect build artifacts. So you can use the path `src/Qir/drops/bin/**` to grab all the binaries available. Locally, this will just get the ones matching your local build, whereas the e2e will get all three platforms.

Reply:
Is there some way to add a dependency on the QIR Runtime build without having a package reference, to ensure that we have these libraries available?

Reply:
No, there really isn't. But this same problem exists for the simulator which you already take a dependency on below. Both the simulator and the QIR Runtime are built during an early step of the build scripts (which in my opinion is always how you should build locally to match the CI pipeline), so this pattern is something we already use and depend on.

That said, it looks like the QIR Runtime is NOT built in the bootstrap.ps1 script, which it should be, since that is how the simulator mitigates this dependency. I'll work on a PR to fix that.

Reply:
#706

Reply:
I was thinking that it might be better to include QIR runtime libraries and headers as part of the `Microsoft.Quantum.Qir.Tools` project rather than the command line project. Since the tools code already assumes that the QIR runtime libraries with specific names exist, I believe it makes sense to distribute the package with the libraries too. It is also more convenient for users of QIR tools because for cases in which the full state simulator is used, they wouldn't have to specify where libraries are located.

Additionally, the CLI11 header is already part of the `Microsoft.Quantum.Qir.Tools` package (see [here](https://github.com/microsoft/qsharp-runtime/blob/1b514cc7e180ffb46b4c6505a90083b86f3db9b4/src/Qir/Execution/Tools/Microsoft.Quantum.Qir.Tools.csproj#L18)).

What do you think @swernli, @ScottCarda-MS?

Reply:
I've removed the package reference and added logic similar to that used to get the simulator dll. Thanks @swernli!

Reply:
Huh, that's an interesting question, @cesarzc. I don't have a strong opinion which package should or shouldn't have it, and since it's so small there isn't much cost to duplicating the dlls into multiple projects. If QIR.Tools does pull the runtime as a dependency then it shouldn't also need to be pulled here, as the dlls and headers will get copied as a transitive dependency, so that would allow some simplification, so I guess it would make sense to have one or the other include them but not both.

Reply:
I've moved the itemgroups for including these resources to the Qir.Tools project, as suggested.